helmfile/values/psp/psp-common.gotmpl

```diff
@@ -0,0 +1,101 @@
+ingressNginx:
+  controller:
+    {{ if .Values.ingressNginx.controller.useHostPort }}
+    hostNetwork: true
+    {{ end }}
+
+prometheus:
+  allowedCapabilities: []
+  allowedHostPaths: []
+  volumes: []
+
+restricted:
+  namespaces:
+    - ingress-nginx
+    - monitoring
+
+  mutation:
+    labelSelector:
+      matchExpressions:
+        - key: app
+          operator: NotIn
+          values:
+            - ciskubebench-exporter
+            - vulnerability-exporter
```

Comment:
Why do they need to be exempt?

Reply:
They are atm running as privileged and required at least root. So one thing that i want to do is to change that they do not run as root etc before doing gatekeeper constraints.